src/current/_includes/releases/v24.2/v24.2.5.md

```diff
@@ -0,0 +1,118 @@
+## v24.2.5
+
+Release Date: November 11, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-5-security-updates">Security updates</h3>
+
+- HBA config entries for LDAP will be evaluated for proper ldap config parameter values, and a valid and complete list of authentication method options is now required to amend HBA settings. [#132748][#132748]
+- You can now authenticate to the DB Console by passing a JWT as the bearer token. [#133534][#133534]
+
+<h3 id="v24-2-5-general-changes">General changes</h3>
+
+- Change the license cockroach is distributed under to the new CockroachDB Software License. [#131707][#131707] [#131956][#131956] [#131961][#131961] [#131983][#131983] [#132011][#132011] [#132013][#132013] [#132012][#132012] [#132015][#132015] [#132014][#132014] [#132016][#132016] [#132054][#132054] [#131970][#131970] [#132056][#132056] [#132801][#132801] [#132704][#132704]
+- The cluster setting `diagnostics.reporting.enabled` is now ignored if the cluster has an Enterprise Trial or Enterprise Free license or if the license cannot be loaded. [#132464][#132464]
+- The new metrics `changefeed.sink_errors` and `changefeed.internal_retry_message_count` allow you to observe the rate of and internal retries for a sink. [#132353][#132353]
+- Added a timer for inner sink client flushes. [#133197][#133197]
+
+<h3 id="v24-2-5-db-console-changes">DB Console changes</h3>
+
+- The DB Console now shows a warning if the cluster is throttled or will be throttled soon due to an expired Enterprise Free or Enterprise Trial license or missing telemetry data. Clusters with an Enterprise license are not throttled. [#132093][#132093]
```

Comment:
```suggestion
- The DB Console now shows a warning if the cluster is throttled or will be throttled soon due to an expired Enterprise Free or Enterprise Trial license, or missing telemetry data. Clusters with an Enterprise license are not throttled. [#132093][#132093]
```

Reply:
Edited your edit to avoid a comma splice.